zulip_botserver/zulip_botserver/server.py

```diff
@@ -65,7 +74,8 @@ def handle_bot(bot):
     message_handler.handle_message(message=event["message"],
                                    bot_handler=restricted_client,
                                    state_handler=state_handler)
-    return "Success!"
+    response_json = json.dumps(dict(response_not_required=True))
+    return response_json
```

Comment:
@timabbott this is would work, although it is a bit redundant. Returning `json.dumps("")` should also work. However we could stick with this as it is more readable.